libs/driver/include/driver/VideoInterface.h

```diff
@@ -72,6 +72,9 @@ class BOOST_SYMBOL_VISIBLE IVideoDriver
     virtual bool IsInitialized() const = 0;
     /// Shall we support OpenGL? (Disabled for tests)
     virtual bool IsOpenGL() const = 0;
+
+    // Fehlermeldung grafisch anzeigen
```

Comment:
Please use English here and triple slash for documentation.
BTW: Any reason why you use C-strings here and not `const std::string&`? Especially as one of the strings you pass in here is already a `std::string`

Reply:
I don't really know what I am doing. This is my first time patching a C++ application.

Reply:
> /usr/local/include/SDL2/SDL_messagebox.h:182:29: note: candidate function not viable: no known conversion from 'const std::string' (aka 'const basic_string<char, char_traits<char>, allocator<char> >') to 'const char *' for 2nd argument

This is the reason. Sorry, it has been too long since I submitted this.